http_client.rst

```diff
@@ -654,6 +655,26 @@ making a request. Use the ``max_redirects`` setting to configure this behavior
         'max_redirects' => 0,
     ]);
 
+Retry Failed Requests
+~~~~~~~~~~~~~~~~~~~~~
+
+Some times, requests failed because of temporary issue in the server or
+because network issue. You can use the
+:class:`Symfony\\Component\\HttpClient\\RetryableHttpClient`
+client to automatically retry the request when it failed.::
+
+    $client = new RetryableHttpClient(HttpClient::create());
+
+The ``RetryableHttpClient`` uses a
+:class:`Symfony\\Component\\HttpClient\\Retry\\RetryDeciderInterface` to
+decide if the request should be retried, and a
+:class:`Symfony\\Component\\HttpClient\\Retry\\RetryBackOffInterface` to
+define the waiting time between each request.
+
+By default, it retries until 3 attemps, the requests responding with a
+status code in (423, 425, 429, 500, 502, 503, 504, 507 or 510) and wait
```

Comment:
Can we say steht like "... for 4xx and 5xx status codes..." or is it not the case for all 400' and 500' codes?

Reply:
it's exactly those http status code (for instance 404 is not retried).
But if it's too verbose, we can reword the sentence `the requests responding with a temporary failure status code`. What do you think?

Reply:
@jderusse when using the HttpClient inside a Symfony app, how can I enable this feature but rely on all its default values? Would the following be enough?

```yaml
    # config/packages/framework.yaml
    framework:
        http_client:
            retry_failed: null
```

Reply:
@javiereguiluz yes both works. 

This is `canBeEnabled` configuration.
default `disabled`, set `null`, `true`, a `enabled: true` subkey, or anyother subkey to enable it.